docs/Admin.md

```diff
@@ -595,6 +598,44 @@ const App = () => (
 );
 ```
 
+### Unplug the <Admin> using the `<AdminContext>` and the `<AdminUI>`
+
+Setting Resources dynamically is still very cumbersome: even if `<Admin>` accepts [a function as child](#declaring-resources-at-runtime), this function can't execute hooks.
+
+So it's impossible, for instance, to have a dynamic list of resources based on a call to the dataProvider (since the dataProvider is only defined after the Admin component renders).
```

Comment:
```suggestion
So it's impossible, for instance, to have a dynamic list of resources based on a call to the `dataProvider` (since the `dataProvider` is only defined after the `<Admin>` component renders).
```